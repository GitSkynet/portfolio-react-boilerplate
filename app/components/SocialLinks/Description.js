import styled from 'styled-components';

const Description = styled.span`
  display: inherit;
  position: absolute;
  width: 1px;
  height: 1px;
  margin: -1px;
  padding: 0;
  border: 0;
  overflow: hidden;
  clip: rect(0 0 0 0);

  @media (min-width: 1280px) {
    display: inline-block;
    height: auto;
    margin: 0 0 0 0.5em;
    overflow: visible;
    position: relative;
    width: auto;
    clip: initial;
    font-size: 1rem;
  }
`;

export default Description;
