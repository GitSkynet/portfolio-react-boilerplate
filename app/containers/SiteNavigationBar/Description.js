import styled from 'styled-components';

const Description = styled.span`
  @media (min-width: 600px) {
    margin-left: 0.5rem;
  }

  @media all and (orientation: landscape) and (min-width: 37.5em) {
    display: none;
  }

  @media all and (orientation: landscape) and (min-height: 725px) {
    display: inline;
  }
`;

export default Description;
