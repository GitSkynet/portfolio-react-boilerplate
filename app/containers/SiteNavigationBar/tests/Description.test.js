import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';

import Description from '../Description';

describe('<Description />', () => {
  it('should render and match the snapshot', () => {
    const renderedComponent = shallow(<Description />);
    expect(toJson(renderedComponent)).toMatchSnapshot();
  });
});
