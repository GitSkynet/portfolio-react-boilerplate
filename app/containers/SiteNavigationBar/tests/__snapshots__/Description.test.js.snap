// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Description /> should render and match the snapshot 1`] = `
@media (min-width:600px) {
  .c0 {
    margin-left: 0.5rem;
  }
}

@media all and (orientation:landscape) and (min-width:37.5em) {
  .c0 {
    display: none;
  }
}

@media all and (orientation:landscape) and (min-height:725px) {
  .c0 {
    display: inline;
  }
}

<span
  className="c0"
/>
`;
